pkg/server/commands/list_objects_shadow.go

```diff
@@ -209,6 +209,11 @@ func (q *shadowedListObjectsQuery) executeShadowModeAndCompareResults(parentCtx
 		return
 	}
 
+	// Don't report on requests that fell through to the old reverse_expand (infinite weight requests)
```

Comment:
Should we move this code above to 194 so that we don't run anything in this method saving resource. Why do shadow when we know we are running the old algorithm.